nvdaHelper/remote/ime.cpp

```diff
@@ -108,12 +108,12 @@ DWORD getIMEVersion(HKL kbd_layout, wchar_t* filename) {
 			// Do not know how to extract version number
 			return 0;
 	}
-	DWORD ver_handle;
+	DWORD ver_handle = 0;
 	DWORD buf_size = GetFileVersionInfoSizeW(filename, &ver_handle);

```

Comment:
apparently the pointer to handle is optional, thus we could just use nullptr?

Reply:
Yes, if we have no need for it.